nodejs18.x/full-stack/{{cookiecutter.project_name}}/samconfig.toml

```diff
@@ -0,0 +1,30 @@
+version = 0.1
+
+[default]
+[default.global.parameters]
+stack_name = "sam-app"
+
+[default.build.parameters]
+cached = true
+parallel = true
+manifest = "backend/package.json"
```

Comment:
Why do we need to set `manifest` option here?

Reply:
I asked for this change. My point was to have 3 different source directories one for each lambda function, and to avoid duplicating the `package.json` in the 3 directories, we can use only one file, and update the `sam build` parameters to add `manifest` option and to refer to the `package.json` path.

Reply:
But, as we still have one directory for all lambda functions, we can remove this option, and to move the `package.json` to `backend\src\handlers`, but it may affect unit testing, and also the instructions to run the testing in Readme.

Reply:
Removed samconfig.toml and updated codeuri and handler paths in template.yaml as suggested.